apps/customer-portal/webapp/src/utils/apiClient.ts

```
@@ -0,0 +1,164 @@
+ */
+export async function authenticatedFetch(
+  url: string,
+  init: RequestInit,
+  callbacks: ApiClientCallbacks,
+): Promise<Response> {
+  const { getToken, signOut, signInSilently } = callbacks;
+
+  const doFetch = async (token: string): Promise<Response> => {
+    const headers = new Headers(init.headers);
+    const authHeaders = addApiHeaders(token);
+    Object.entries(authHeaders).forEach(([k, v]) => headers.set(k, v));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)